dhis-2/dhis-web/dhis-web-dataentry/src/main/java/org/hisp/dhis/de/action/LoadFormAction.java

```diff
@@ -472,6 +477,80 @@ private void getSectionForm( Collection<DataElement> dataElements, DataSet dataS
         }
     }
 
+    private void processSectionForUserOrdering( Section section )
```

Comment:
Looks good. Could we move at least part of these two methods to a utility class and add unit tests for them?

Reply:
sure.